apps/src/flowlab/flow/askAi.ts

```diff
@@ -1,25 +1,33 @@
 import {postAichatCompletionMessage} from '@cdo/apps/aichat/aichatApi';
-import {PendingChatMessage, AichatContext} from '@cdo/apps/aichat/types';
+import {
+  PendingChatMessage,
+  AichatContext,
+  ChatAsset,
+} from '@cdo/apps/aichat/types';
 import {EMPTY_AI_CUSTOMIZATIONS} from '@cdo/apps/aichat/views/modelCustomization/constants';
 import {Role} from '@cdo/apps/aiComponentLibrary/chatMessage/types';
 import {
   AiChatModelIds,
   AiInteractionStatus as Status,
 } from '@cdo/generated-scripts/sharedConstants';
 
-export default async function askAi(message: string) {
+export default async function askAi(
+  message: string,
+  channelId?: string,
+  assets?: ChatAsset[]
+) {
   const newUserMessage: PendingChatMessage = {
     role: Role.USER,
     status: Status.UNKNOWN,
     chatMessageText: message,
-    assets: undefined,
     timestamp: Date.now(),
+    assets,
   };
 
   const aichatContext: AichatContext = {
     currentLevelId: null, // this.currentLevelId ? parseInt(this.currentLevelId) : null,
     scriptId: null, // this.scriptId || null,
```

Comment:
Technically you could also grab these from Redux (in AiNode.tsx) and pass them in like `channelId`

Reply:
Good call.  Done.